sdk/formrecognizer/azure-ai-formrecognizer/migration-guide.md

```diff
@@ -0,0 +1,590 @@
+
+## Table of contents
+- [Migration benefits](#migration-benefits)
+- [Important changes](#important-changes)
+    - [Instantiating clients](#instantiating-clients)
+    - [Analyze documents](#analyze-documents)
+      - [Using a prebuilt Model](#using-a-prebuilt-model)
+      - [Using a layout model](#using-a-layout-model)
+      - [Using custom model](#using-a-custom-model)
+    - [Manage models](#manage-models)
+- [Additional samples](#additional-samples)
+
+## Migration benefits
+
+A natural question to ask when considering whether to adopt a new version of the library is what the benefits of
+doing so would be. As Azure Form Recognizer has matured and been embraced by a more diverse group of developers, 
+we have been focused on learning the patterns and practices to best support developer productivity and add value to our 
+customers.
+
+To improve the development experience and addressing the consistent feedback across the Form Recognizer SDK, this new 
+version of the library replaces the previously existing clients `FormRecognizerClient` and `FormTrainingClient` with
+`DocumentAnalysisClient` and the `DocumentModelAdministrationClient` that provide unified methods for 
+analyzing documents and providing support for the new features added by the service in 
+API version `2021-09-30-preview` and later.
+
+The below table describes the relationship of each client and its supported API version(s):
+
+|API version|Supported clients
+|-|-
+|2021-09-30-preview | DocumentAnalysisClient and DocumentModelAdministrationClient
+|2.1 | FormRecognizerClient and FormTrainingClient
+|2.0 | FormRecognizerClient and FormTrainingClient
+
+The newer Form Recognizer client library also provides the ability to share in some improvements made to the Azure development experience, such as:
+
+- A unified method, `beginAnalyzeDocument` and `beginAnalyzeDocumentFromUrl`, for analyzing text and structured data from documents.
+This method uses a `modelId` parameter for specifying the type of analysis to perform. 
+The newly introduced method return type `AnalyzeResult` removes hierarchical dependencies between the previously known `FormElements`
+and move them to a more top level and easily accessible position such as `AnalyzeResult.tables` instead of `RecognizedForm.pages.tables`.
+The service has further matured to define cross-page elements by using the `BoundingRegion` model and by specifying the content and span information on document fields.
+- A unified return type `DocumentModel` indicating the document types the model can analyze and the specific fields it can analyze along with the estimated confidence for each field.
+- Specifying a modelId instead of the generated GUID when creating custom models, along with an optional description. See [here][service_supported_models], for the supported model types.
```

Comment:
```suggestion
- Specifying a modelId instead of the generated GUID when creating, composing or copying models, along with an optional description. See [here][service_supported_models], for the supported model types.
```